Adyen/Core/Core Protocols/PartialPaymentDelegate.swift

```diff
@@ -12,21 +12,26 @@ public protocol PartialPaymentDelegate: AnyObject {
     /// Invoked when the payment component needs a balance check call to be done by the merchant.
     ///
     /// - Parameters:
+    ///   - component: The current component.
     ///   - data: The data supplied by the payment component.
     ///   - completion: The completion closure called when the balance is checked.
     func checkBalance(with data: PaymentComponentData,
+                      component: Component,
                       completion: @escaping (Result<Balance, Error>) -> Void)
 
     /// Invoked when the payment component needs a partial payment order object.
     ///
     /// - Parameters:
+    ///   - component: The current component.
     ///   - completion: The completion closure called when the order object.
-    func requestOrder(_ completion: @escaping (Result<PartialPaymentOrder, Error>) -> Void)
+    func requestOrder(in component: Component,
```

Comment:
not sure about the `in` 🤔

Reply:
Hmm, what would you suggest?

Reply:
just `func requestOrder(_ component: Component,` is fine I guess